darwin/dataset/upload_manager.py

```diff
@@ -402,15 +509,53 @@ def _upload_file(
 
 
 class UploadHandlerV2(UploadHandler):
-    def __init__(self, dataset: "RemoteDataset", local_files: List[LocalFile]):
-        super().__init__(dataset=dataset, local_files=local_files)
+    def __init__(
+        self,
+        dataset: "RemoteDataset",
+        local_files: List[LocalFile],
+        multi_file_items: Optional[List[MultiFileItem]] = None,
+    ):
+        super().__init__(
+            dataset=dataset,
+            local_files=local_files,
+            multi_file_items=multi_file_items,
+        )
 
     def _request_upload(self) -> Tuple[List[ItemPayload], List[ItemPayload]]:
         blocked_items = []
         items = []
         chunk_size: int = _upload_chunk_size()
-        for file_chunk in chunk(self.local_files, chunk_size):
-            upload_payload = {"items": [file.serialize_v2() for file in file_chunk]}
+        single_file_items = self.local_files
+        upload_payloads = []
+        if self.multi_file_items:
+            upload_payloads.extend(
+                [
+                    {
+                        "items": [file.serialize_v2() for file in file_chunk],
+                        "options": {"ignore_dicom_layout": True},
+                    }
+                    for file_chunk in chunk(self.multi_file_items, chunk_size)
+                ]
+            )
+            local_files_for_multi_file_items = [
+                file
+                for multi_file_item in self.multi_file_items
+                for file in multi_file_item.files
+            ]
+            single_file_items = [
+                file
+                for file in single_file_items
+                if file not in local_files_for_multi_file_items
+            ]
+
+        upload_payloads.extend(
+            [
+                {"items": [file.serialize_v2() for file in file_chunk]}
+                for file_chunk in chunk(single_file_items, chunk_size)
+            ]
+        )
```

Comment:
We do it this way so that if in the future, we want to develop a more complex multi-file item upload feature: A single `push()` operation is capable of uploading single-file items and multi-file items

Reply:
I think it is safe to default `ignore_dicom_layout` to true.